src/PrestaShopBundle/Service/DataProvider/Marketplace/ApiClient.php

```diff
@@ -45,6 +45,7 @@ public function __construct(
         $this->setIsoLang($isoLang)
             ->setIsoCode($isoCode)
             ->setVersion(_PS_VERSION_)
+            ->setSslVerification(!_PS_MODE_DEV_)
```

Comment:
As discussed with @julienbourdeau, we're going to introduce a configuration parameter to handle this case without having it hardwired to a constant.